### DISCUSSION TITLE
11.85.0.32 EARLY RELEASE

### DISCUSSION THREAD

**Discussion by ghost** [rodit/SnapMod#195] [General]
built by me hf

[11.85.0.32.zip](https://github.com/rodit/SnapMod/files/8998404/11.85.0.32.zip)

just add .json is folder

**Comment by RSR1337** [comment]
> built by me hf
> 
> [11.82.0.32.zip](https://github.com/rodit/SnapMod/files/8889210/11.82.0.32.zip)
> 
> just add .json is folder

Works fine, thank you

**Reply by Xulpha** [reply]
How to use? Sorry I tried renaming to .JSON and adding to the location, said it was supported but snap wouldn't open

**Comment by rodit** [comment]
I'm assuming the notification content and custom chat menus don't work (unless you fixed those)? I have updated to 11.82.0.32 and added story pinning, just not pushed it yet so you'll see it soon ish.

**Reply by ghost** [reply]
i am only running the default search hence the early. i have it automated. via apk mirror 🗡️

**Reply by rodit** [reply]
Nice, it's quite likely some stuff won't work without modifying snap.ds though

**Comment by ghost** [comment]
bumped version

**Reply by flawnn** [reply]
Hey dude, wanna have chat? Where can I contact you. (I am not a Snapchat lawyer 😆 )

**Comment by ghost** [comment]
working best with https://github.com/rodit/SnapMod/releases/tag/v1.8.2 cuz story issues!

**Comment by ghost** [comment]
[11.84.0.16.zip](https://github.com/rodit/SnapMod/files/8974023/11.84.0.16.zip)

Updated 👍

**Reply by flawnn** [reply]
How can I create my own bindings?

**Reply by authorisation** [reply]
> How can I create my own bindings?

Use [dexsearch](https://github.com/rodit/dexsearch) and the snap.ds schematic

**Comment by ghost** [comment]
[11.85.0.32.zip](https://github.com/rodit/SnapMod/files/8998384/11.85.0.32.zip)

Updated 🦖

**Reply by cgallagher-dev** [reply]
sorry this is probably dumb but how do i use this? what directory do i copy the json into, and do i have to re-name it? thanks!

**Reply by us3rname101** [reply]
Hi, you need to unzip it and rename the json to 84619.json, it's the build number, you can find it on apkmirror. Then you move it to /data/data/xyz.rodit.snapmod/files/[build].json or /Android/data/xyz.rodit.snapmod/files/[build].json

**Reply by RickyXO2** [reply]
Hi. I'm not sure what exactly I'm doing wrong but I unzipped the file, renamed the .json file to 84619, moved it to /data/data/xyz.rodit.snapmod/files/ & Snapchat just freezes up. I've tried clearing the cache and even the data for both apps but still can't figure it out. Is there something else that I'm missing?

**Reply by us3rname101** [reply]
Odd. Does SnapMod say it found those mappings? What SnapMod version are you running? What version is your Snapchat?

**Reply by RickyXO2** [reply]
![Screenshot_20220704-082331903](https://user-images.githubusercontent.com/106756003/177164038-c5404da8-ab6f-4c19-b464-3e825b72be05.jpg)

I believe it can recognize the mappings but this pops up whenever I open the app.

**Reply by us3rname101** [reply]
You might be on an old version, try 1.8.2. Theres 1.8.3 but it breaks stories somewhat.

**Reply by RickyXO2** [reply]
I downgraded from 1.8.3 to 1.8.2 but I still have the same results. I also tried clearing cache & rebooting as good measure but no luck.

**Reply by us3rname101** [reply]
Then I have no idea, sorry

**Reply by RickyXO2** [reply]
I finally got it to work. Not even sure how.
